htcanalyze/main.py

```diff
@@ -77,7 +83,7 @@ def check_for_redirection(self):
 
 
 ###############################
-GlobalServant = GlobalPlayer()
+GlobalStream = GlobalStreams()
```

Comment:
I renamed this in the other WIP PR to CheckRedirection, what do you think of this ?

Reply:
And then RedirectionChecker = CheckRedirection()

Reply:
My suggestion is to remove the entire class and deal with this in `main()`, but if you've decided to retain it, then I suggest `streams = StandardStreams()` (or, if you prefer your naming convention, `redirection_checker = CheckRedirection()`.

`StandardStreams()` as the class name, because it contains all the data about the standard streams (`stdin`, `stdout`, `stderr`) that you'll use. `streams` as the instance name, because it is valid snake case that would be used as `streams.redirect_stdout`, etc.